tests/pages/package-edit.js

```diff
@@ -32,8 +39,12 @@ import Datepicker from './datepicker';
 }
 
 @interactor class PackageEditDropDownMenu {
+  addToHoldings = new Interactor('.tether-element [data-test-eholdings-package-add-to-holdings-action]');
+  removeFromHoldings = new Interactor('.tether-element [data-test-eholdings-package-remove-from-holdings-action]');
   clickRemoveFromHoldings = clickable('.tether-element [data-test-eholdings-package-remove-from-holdings-action]');
   clickAddToHoldings = clickable('.tether-element [data-test-eholdings-package-add-to-holdings-action]');
+  isAddToHoldingsPresent = isPresent('.tether-element [data-test-eholdings-package-add-to-holdings-action]');
```

Comment:
I believe we can just use `expect(interactor.addToHoldings.isVisible).to.equal(true)`

Reply:
I did try that --  it is cleaner and it works for the true case. But If I have the case to check that the menu item is not present `expect(PackageEditPage.dropDownMenu.addToHoldings.isVisible).to.equal(false);`
I get 
` Error: unable to find ".tether-element [data-test-eholdings-package-add-to-holdings-action]"`
So not sure how to handle that within an interactor like that